utils/merge.py

```
@@ -131,27 +218,28 @@ def make_book_model(asin, original_path) -> Book:
         title=title,
         asin=asin,
-        short_desc=metadata['description'],
-        long_desc=metadata['summary'],
+        short_desc=metadata["description"],
+        long_desc=metadata["summary"],
         release_date=datetime.strptime(
-            metadata['releaseDate'], '%Y-%m-%dT%H:%M:%S.%fZ'),
-        publisher=metadata['publisherName'],
-        lang=metadata['language'],
+            metadata["releaseDate"], "%Y-%m-%dT%H:%M:%S.%fZ"
+        ),
```

Ruff (0.15.0)

[warning] Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)